### DISCUSSION TITLE
guesstimation of time needed in token / s for 1 laptop vs 2 laptops in MPI mode.

### DISCUSSION THREAD

**Discussion by hiqsociety** [ggml-org/llama.cpp#3161] [Q&A] [answered]
guesstimation of time needed in token / s for 1 laptop vs 2 laptops in MPI mode.

if 1 laptop is 10t/s, 2 laptops = 18t/s? is this a good guesstimation?

the more powerful the gpu card, the more powerful the CPU is needed and more ram etc too. so i was wondering if it's just better to buy multiple devices more maybe at 1/3 of the price than just buy 1 powerful machine that will most probably drop its price soon.

33b model is what i am to run "comfortably" (on laptops) but the gpus for laptops are more expensive than desktop (i dont want the extra heat and power consumption and worst, non portability).

what do u guys think? 2 laptops instead of 1?

**Comment by KerfuffleV2** [comment]
I don't know the answer to your question but only 10% overhead sounds absolutely amazing. I'd be surprised if the real number wasn't at least 30-40%.

**Reply by staviq** [reply, marked as answer]
#3099

**Comment by hiqsociety** [comment]
@staviq @KerfuffleV2 how to calculate the overhead so as to have the right set up i wonder?

it also makes me wonder if it's a viable option for multi datacenter processing at what kind of network overhead.

how to calculate it?
